service/src/main/java/uk/gov/hmcts/reform/fpl/service/InboxLookupService.java

```diff
@@ -8,27 +9,33 @@
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 import uk.gov.hmcts.reform.fpl.model.Solicitor;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 
 import static uk.gov.hmcts.reform.fpl.config.LocalAuthorityEmailLookupConfiguration.LocalAuthority;
 
 @Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class InboxLookupService {
     private final LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration;
-    private final String fallbackInbox;
+    private final FeatureToggleService featureToggleService;
 
-    @Autowired
-    public InboxLookupService(LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration,
-                              @Value("${fpl.local_authority_fallback_inbox}") String fallbackInbox) {
-        this.localAuthorityEmailLookupConfiguration = localAuthorityEmailLookupConfiguration;
-        this.fallbackInbox = fallbackInbox;
-    }
+    @Value("${fpl.local_authority_fallback_inbox}")
+    private String fallbackInbox;
 
-    public String getNotificationRecipientEmail(final CaseData caseData) {
-        return localAuthorityEmailLookupConfiguration.getLocalAuthority(caseData.getCaseLocalAuthority())
+    public List<String> getNotificationRecipientsEmails(final CaseData caseData) {
+        List<String> recipients = new ArrayList<>();
+        localAuthorityEmailLookupConfiguration.getLocalAuthority(caseData.getCaseLocalAuthority())
             .map(LocalAuthority::getEmail)
             .filter(StringUtils::isNotBlank)
-            .orElseGet(() -> getSolicitorOrFallbackEmail(caseData.getSolicitor()));
+            .ifPresent(recipients::add);
+
+        if (recipients.isEmpty() || featureToggleService.isSendToSolicitorEnabled(caseData.getCaseLocalAuthority())) {
+            recipients.add(getSolicitorOrFallbackEmail(caseData.getSolicitor()));
```

Comment:
this is wrong, assume that there is default email found in  line 29 and toggle says add solicitor email, but there is not solicitor email present. I dont think so that we should add fallback email. Fallback email should be used if no other emails available

Reply:
fixed (i think)